python/setup.py

```diff
@@ -7,6 +7,21 @@
 
 import versioneer
 
+if "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE" in os.environ:
+    orig_get_versions = versioneer.get_versions
+
+    version_override = os.environ.get(
+        "RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE", ""
+    )
+
+    def get_versions():
+        data = orig_get_versions()
+        if version_override != "":
+            data["version"] = version_override
+        return data
+
+    versioneer.get_versions = get_versions
```

Comment:
Should the `setup(..., version=..., ...)` be modified to call this function instead of reading the environment variable again? I think that this monkey-patch will make `versioneer.get_version()` return the right short version. Ref:

https://github.com/python-versioneer/python-versioneer/blob/83f20fdf886df7a6089fca3bb7fc3c33198bd629/src/get_versions.py#L90

Like:
```python
setup(
    name="rmm" + os.getenv("RAPIDS_PY_WHEEL_CUDA_SUFFIX", default=""),
    version=versioneer.get_version(),
)
```